x-pack/plugins/visualization_lens/public/types.ts

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface EditorFrameAPI {
+  render: (domElement: Element) => void;
+  registerDatasource: (datasource: Datasource<unknown>) => void;
+  registerVisualization: (visualization: Visualization<unknown>) => void;
+}
+
+export interface EditorFrameState {
+  visualizations: { [id: string]: object };
```

Comment:
Those are also `unknown`s, right?

Reply:
We know it should be an object, but not what the object looks like, so it makes sense to use the least common denominator here

Reply:
Do we even need this interface here? This is an implementation detail of the editor frame and nothing other plugins will use, right? I think we should just remove it and address this when setting up the state handling in the editor frame